docs/design-documents/tools/cmake.md

```diff
@@ -96,6 +92,5 @@ A user create own CMake file to configure an application, also with `mbed_app.js
 2. Parse the application configuration
```

Comment:
Do you want to carry on using `1` for numbered bullet points so you do not need to remember what the next number is?

Reply:
yes

Reply:
will fix